### ISSUE TITLE
Option --plot-prefix not available

### ISSUE THREAD

**Issue by Lehnt** [Hoohm/CITE-seq-Count#29]
Hi, I get this type of error when running CITE-seq-Count :

`root:No local minima was accepted. Recommend checking the plot output and counts per local minima (requires `--plot-prefix` option) and then re-running with manually selected threshold (`--set-cell-number` option)`

However, when I try to add a --plot-prefix to the command line, it says there is no such option (with version 1.4.1). Is this behavior expected ?

**Comment by Hoohm** [comment]
Hello,

this is odd because what you are getting is the error from umi_tools when it can't find a good cell barcode whitelist.

This should not appear. Could you show me your command line?

**Comment by Lehnt** [comment]
Hi, the command line I use is 
CITE-seq-Count -R1 R1.fastq.gz \
	-R2 R2.fastq.gz \
	-t TAG_LIST.csv \
	-cbf 1 -cbl 16 \
	-umif 17 -umil 26 -o Result.tsv \
        -wl $barcodes --expected_cells 2000

Python 3.7.1
CITE-seq-Count v1.4.1
UMI-tools version: 0.5.5

**Comment by Lehnt** [comment]
Correct, it still runs until the end. So I guess the best way to run CITE-seq-Count in this case is running first UMI-tools and find the optimal number of cells, and then CITE-seq-Count, or do you suggest another approach ?

**Comment by Hoohm** [comment]
I'm thinking about it.
I could also make it interactive and hang when this occurs, make the plots and ask you to enter the number of cells you select in the end. This entails a few issues where interaction is not possible, ie a cluster run.

Running umi_tools first is not always gonna work because you can't access the `--set-cell-number` option through CITE-seq-Count for now. If you try, let me know how it goes.

Depending on the sequencing quality, you can also leave it as is, most of the reads won't be lost to the 1 mutated barcode list anyways.

**Comment by Hoohm** [comment]
@joonlee3 
You can disregard it. Basically what's happening is that umi_tools was not able to decide which cells are "true" cells based on the top N given and their respective read distribution.

umi_tools is expecting mRNA counts which have a different distribution than CITE-seq/HTO data. So I'm not surprised that it can't always find a good list of true cells.

I'll find a better way to handle the error so that people are not confused by the message.

**Comment by r-mvl** [comment]
I am having the same problem @Hoohm . Although in my case it never finishes. I let it run for > 12h and it's still "running", while the error log file gives the same message as above. Any idea how to fix this?

**Comment by Hoohm** [comment]
Hello @r-mvl 
Could you please try and use the develop branch, it should fix the issue of never finishing the umi correction. If it still runs for too long, please use the option `--no_umi_correction` to deactivate the umi correction while I fix it for the next release.

This is how you install and use the develop branch:

```
git clone https://github.com/Hoohm/CITE-seq-Count.git
cd CITE-seq-Count
git checkout develop
python3.6 setup.py develop
```

If you type: `CITE-seq-Count --version` you should get 1.4.2

**Comment by Hoohm** [comment]
Fixed in 1.4.2. Closing this
